source/content/partials/drush-compatibility.md

```diff
@@ -8,36 +8,30 @@ tags: [--]
 reviewed: ""
 ---
 
-Drush only supports one install method. Your Drupal site is required to be built with Composer and your Drush must be compatible and listed as a dependency. 
-
-See the [Drush Drupal Compatibility chart](https://www.drush.org/latest/install/#drupal-compatibility) for version compatibility information. Consider the following information when determining which Drush version is best suited for your site:
-
-* Drush 10 is available with the [addition of the  `pantheon.yml` file](#configure-drush-version), or for [site-local installation](#site-local-drush-usage). It requires Drupal 9 or higher, [Composer](/guides/composer/), and PHP 7.1 or higher.
+<Alert title="Note" type="info">
 
-* [Drush 9](https://docs.drush.org/en/9.x/) and Drush 10 only work on Drupal 8.4 and higher.
+We highly recommend running Drush version 8.3.2 or higher when running Drush locally.
 
-* [Drush 8](https://docs.drush.org/en/8.x/) Drush 8 is the only recommended version to use in your `pantheon.yml` file. It is compatible with Drupal 7.
+__Is that still true?__
 
-* Drush 5 and Drush 7 are available on Pantheon if needed. However, both versions are listed as [unsupported](https://docs.drush.org/en/8.x/install/#drupal-compatibility) by the Drush maintainers, and should be avoided unless absolutely necessary.
 
-<Alert title="Note" type="info">
+</Alert>
 
-We highly recommend running Drush version 8.3.2 or higher when running Drush locally.
+See the [Drush Drupal Compatibility chart](https://www.drush.org/latest/install/#drupal-compatibility) for version compatibility information. Consider the following information when determining which Drush version is best suited for your site.
 
-</Alert>
 
 ### Drush Requirements and Compatibility
 
-| Drush Version  | PHP Version | End of Life     | Drupal 7| Drupal 8 | Drupal (Latest Version)
-| ------------- |-------------|-------------|-------------   |------------- |-------------  |
+| Drush Version  | PHP Version | End of Life     | Drupal 7| Drupal 8 ⚠️ <Popover title="Drupal 8 EOL" content="Drupal 8 is past its end of life date. While a version of Drush may function with Drupal 8, do not build for the future on that combination" /> | Drupal 9+
```

Comment:
That entire popover wound up being bold, which I think it should not be?

Reply:
hmmm, @mel-miller, this whole chart feels like it could use a design eye. What do you think of it? https://pr-9010-documentation.appa.pantheon.site/guides/drush/drush-versions

Reply:
@stevector — Yeah it definitely needs some design TLC. Should we circle back later and fix it up, or hold for design?

Reply:
Since we link to [this info](https://www.drush.org/12.x/install/#drupal-compatibility), I think we do not need this table.